File: Datas/本地数据表/CreateLua/GameLevelMonsterEntity.lua
GameLevelMonsterEntity = { Id = 0, GameLevelId = 0, Grade = 0, RegionId = 0, SpriteId = 0, SpriteCount = 0, Exp = 0, Gold = 0, DropEquip = "", DropItem = "", DropMaterial = "" }

--这句是重定义元表的索引，就是说有了这句，这个才是一个类
GameLevelMonsterEntity.__index = GameLevelMonsterEntity;

function GameLevelMonsterEntity.New(Id, GameLevelId, Grade, RegionId, SpriteId, SpriteCount, Exp, Gold, DropEquip, DropItem, DropMaterial)
    local self = { }; --初始化self
    setmetatable(self, GameLevelMonsterEntity); --将self的元表设定为Class

    self.Id = Id;
    self.GameLevelId = GameLevelId;
    self.Grade = Grade;
    self.RegionId = RegionId;
    self.SpriteId = SpriteId;
    self.SpriteCount = SpriteCount;
    self.Exp = Exp;
    self.Gold = Gold;
    self.DropEquip = DropEquip;
    self.DropItem = DropItem;
    self.DropMaterial = DropMaterial;

    return self;
end

File: Datas/本地数据表/Create/GameLevelMonsterEntity.cs

//===================================================
//作    者：边涯  http://www.u3dol.com  QQ群：87481002
//创建时间：2017-10-24 15:58:08
//备    注：此代码为工具生成 请勿手工修改
//===================================================
using System.Collections;

/// <summary>
/// GameLevelMonster实体
/// </summary>
public partial class GameLevelMonsterEntity : AbstractEntity
{
    /// <summary>
    /// 游戏关卡Id
    /// </summary>
    public int GameLevelId { get; set; }

    /// <summary>
    /// 难度等级
    /// </summary>
    public int Grade { get; set; }

    /// <summary>
    /// 区域Id
    /// </summary>
    public int RegionId { get; set; }

    /// <summary>
    /// 精灵Id
    /// </summary>
    public int SpriteId { get; set; }

    /// <summary>
    /// 精灵数量
    /// </summary>
    public int SpriteCount { get; set; }

    /// <summary>
    /// 掉落经验
    /// </summary>
    public int Exp { get; set; }

    /// <summary>
    /// 掉落金币
    /// </summary>
    public int Gold { get; set; }

    /// <summary>
    /// 掉落装备
    /// </summary>
    public string DropEquip { get; set; }

    /// <summary>
    /// 掉落道具
    /// </summary>
    public string DropItem { get; set; }

    /// <summary>
    /// 掉落材料
    /// </summary>
    public string DropMaterial { get; set; }

}
